permission_handler_html/lib/web_delegate.dart

```diff
@@ -103,8 +98,8 @@ class WebDelegate {
       // The manual stop action is then needed here for to stop the automatic
       // recording.
 
-      if (mediaStream.active!) {
-        final videoTracks = mediaStream.getVideoTracks();
+      if (mediaStream?.active ?? false) {
```

Comment:
As with the microphone permissions, we should `return false;` earlier in my opinion